src/assets/scss/andculturecode-javascript-react-components.scss

```diff
@@ -0,0 +1,28 @@
+/**
+ * CONTENTS
+ *
+ * SETTINGS
+ * Global variables and maps that are used throughout the project
+ *
+ * TOOLS
+ * Mixins and functions used throughout the application for better reuse
+ *
+ * GENERIC
+ * Normalization and resetting of all html elements
+ *
+ * ELEMENTS
+ * General elements such as headings, paragraphs, anchors, etc...
+ *
+ * OBJECTS
+ * Class-based selectors which define undecorated design patterns, for example media object known from OOCSS.
+ *
+ * COMPONENTS
+ * All items that describe the physical attributes like buttons, sliders, cards, badges, etc...
+ *
+ * UTILITIES
+ * Utilities and helper classes with ability to override anything which goes before in the triangle, eg. hide helper class
+ */
+
+@import "0-vendor/__vendor", "1-settings/__settings", "2-tools/__tools",
```

Comment:
Nitpick - I prefer breaking out an `@import "partial";` per line so it is easier to alphabetize

Reply:
ah, i like that alternative being when comma-delimited it gets auto-formatted 👍